app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list[dict[any]], cafe: Cafe) -> str:
```

Comment:
It's a good practice to add type annotation for the return type of the function. In this case, the function is expected to return a string.